sites/fast-creator/app/configs/map-native-element-definition-to-json-schema.ts

```diff
@@ -0,0 +1,27 @@
+import { mapWebComponentDefinitionToJSONSchema } from "@microsoft/fast-tooling";
+import {
+    WebComponentDefinition,
+    WebComponentDefinitionTag,
+} from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { nativeElementDefinitions } from "@microsoft/site-utilities";
+
+const nativeElementSchemas: { [key: string]: any } = {};
+
+function mapToJSONSchemas(): void {
+    Object.entries(nativeElementDefinitions).forEach(
+        ([, definition]: [string, WebComponentDefinition]) => {
```

Comment:
nit: leading comma?

Reply:
It has to be there in order to do a non-capture on the first item in the array